stress/common/src/main/java/alluxio/stress/client/ClientIOParameters.java

```diff
@@ -93,6 +93,11 @@
           + "file will be round-robin across these number of workers.")
   public int mWriteNumWorkers = 1;
 
+  @Parameter(names = {"--profile-agent"},
```

Comment:
Since this is common, and may be common to more, can we just put this in BaseParameters?